modules/op-adding-triggers.adoc

```diff
@@ -63,7 +63,7 @@ spec:
   - apiVersion: tekton.dev/v1beta1
     kind: PipelineRun
     metadata:
-      name: build-deploy-$(tt.params.git-repo-name)-$(uid)
+      generateName: build-deploy-$(tt.params.git-repo-name)-
```

Comment:
Line 134 needs to be updated to `https://github.com/openshift/pipelines-tutorial/raw/{pipelines-ver}/03_triggers/03_trigger.yaml`